### PR TITLE
Add `hashrate.no` extension

### PR DESCRIPTION
**Pull request by xmok**
## Description

Seeing the recent PRs for crypto related extensions, here's one I made in July for some https://hashrate.no/ functions.

## Screencast

https://github.com/user-attachments/assets/15a27795-1818-49c7-8be0-da2c3569123a

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/hashrate-no/package.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "hashrate-no",
+  "title": "Hashrate.no",
```
Comment (pernielsentikaer) [inline extensions/hashrate-no/package.json:4]: Can we call it Hashrate instead?

File: extensions/hashrate-no/package.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      "type": "password",
+      "required": true,
+      "placeholder": "XXX...XX",
+      "description": "API Key from Hashrate.no"
+    }
+  ],
+  "commands": [
+    {
+      "name": "coins",
+      "title": "Coins",
```
Comment (pernielsentikaer) [inline extensions/hashrate-no/package.json:26]:
```suggestion
      "title": "Coins",
      "subtitle": "Hashrate",
```

File: extensions/hashrate-no/package.json [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      "description": "Crypto Coins with Pricing and Estimates",
+      "mode": "view"
+    },
+    {
+      "name": "gpu_estimates",
+      "title": "GPU Estimates",
```
Comment (pernielsentikaer) [inline extensions/hashrate-no/package.json:32]:
```suggestion
      "title": "GPU Estimates",
      "subtitle": "Hashrate",
```

**Comment by xmok** [comment]
I would be on board with renaming but the only factor is that "Hashrate" or "Hash rate" is also a term in blockchain so I wouldn't want one to conflate "Hashrate the term" and "Hashrate the API".

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**